src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -135,7 +136,10 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
     reValidateMode: "onChange",
     validationSchema: validationSchema,
     validationContext: { editing: !!editingRoom },
-    defaultValues,
+    defaultValues: {
+      ...defaultValues,
+      url: defaultValues.url ?? venueInsideUrl(venueId),
```

Comment:
I don't think I understand where does venueId is coming from

Reply:
On line 45 in RoomsForm component:
```typescript
const venueId = useVenueId();
```

Then it's passed to `RoomInnerForm` on line 102:
```typescript
<RoomInnerForm
        venueId={venueId}
        venue={venue}
        editingRoom={room}
        editingRoomIndex={queryRoomIndex}
      />
```

And then it gets to the `defaultValues`.